unit-tests/src/test/scala/scala/scalanative/unsafe/CFuncPtrOpsSuite.scala

```diff
@@ -14,4 +14,10 @@ object CFuncPtrOpsSuite extends tests.Suite {
       randFunc.asInstanceOf[CFuncPtr1[Int, Int]] // wrong arity
     }
   }
+
+  test("CFuncPtr.toPtr casts to Ptr[Byte]") {
+    assertTrue {
+      randFunc.toPtr.isInstanceOf[Ptr[Byte]]
```

Comment:
Testing that the result of a method that returns a `Ptr[Byte]` is an instance of a `Ptr[Byte]` is redundant, and does not actually anything beside the fact that the method does not crash.

End-to-end tests for conversions should always test that we can actually *do* something meaningful with the result.

This test does not prevent me from implementing `toPtr` as always returning `unsafe.alloc[Byte]`. A meaningful test would invalidate that implementation.